tests/testthat/spacing/spacing_comma2-out.R

```diff
@@ -0,0 +1,2 @@
+call(arg, , more_args)
+a[, , drop = FALSE]
```

Comment:
Do we need a space before the first comma?

Reply:
I'd say the convention is not to have one, but the example is poorly chosen. You are more likely to see something like this:
```r
a[1,, drop = FALSE]
```

Reply:
This feels sufficiently rare, let's wait for a real use case.